### ISSUE TITLE
Include slide share tag

### ISSUE THREAD

**Issue by octante** [tlksio/front#96]
As commented saturday and today again, include new field in talk form to add slides from slide share.

**Comment by repejota** [comment]
:+1:  Agree but I would like to see that optional, we should keep the process to add talks easy and fast to increase the ratio of new talks on the database. At least in the beggining.

**Comment by octante** [comment]
Yes, totally agree, for me is some useful data for users, but totally optional.

**Comment by repejota** [comment]
Changed a little bit the title. Supporting only slide share would not enough. Maybe we can support other places like:

* SlideShare 
* SpeakerDeck
* More?
